sympy/functions/special/error_functions.py

```diff
@@ -1409,7 +1415,7 @@ def fdiff(self, argindex=1):
     def _eval_conjugate(self):
         z = self.args[0]
         # Exclude values on the branch cut (-oo, 0)
-        if not (z.is_real and z.is_negative):
+        if not z.is_negative:
```

Comment:
Shouldn't this be `z.is_extended_negative` ?

Reply:
Yes, I've changed it to that